### ISSUE TITLE
wpt.fyi should default to showing experimental results

### ISSUE THREAD

**Issue by thejohnjansen** [web-platform-tests/wpt.fyi#723]
I cannot remember exactly who stated this as an action, but during our meeting in TPAC I took a note to log this issue. The idea being that webdevs care more about what is coming than what is presently here.

**Comment by lukebjerring** [comment]
We have a feature for this #695 (waiting on prod push #720).
Currently enabled on https://staging.wpt.fyi

**Comment by lukebjerring** [comment]
Note that the reason we haven't already done this in production is that we have 2 problems:
- Edge is never "aligned" with the other 3 products (to compare apples-and-apples) 
- Comparing Edge stable (from Sauce) still isn't apples and apples, and Edge Stable results are misbehaving lately.

@thejohnjansen, where are you at with grabbing a SHA from the announcer before you execute?

**Comment by gsnedders** [comment]
Who do we believe the current default view is helping? How many of our users are primarily concerned with stable releases, and how often?

It seems unlikely we're going to get regular Edge insider results any time soon (I imagine @thejohnjansen is largely busy with non-EdgeHTML work!), so maybe it's time to reconsider just mixing Edge stable with everything else, despite Sauce's issues.

**Comment by lukebjerring** [comment]
Agreed. Launching the "aligned runs with all experimental browsers except edge" feature now.
